pkgs/applications/window-managers/openbox/default.nix

```diff
@@ -22,15 +27,37 @@ stdenv.mkDerivation rec {
     pango imlib2
   ];
 
-  pythonPath = with python2.pkgs; [
+  pythonPath = with python3.pkgs; [
     pyxdg
   ];
 
-  src = fetchurl {
-    url = "http://openbox.org/dist/openbox/${pname}-${version}.tar.gz";
-    sha256 = "1xvyvqxlhy08n61rjkckmrzah2si1i7nmc7s8h07riqq01vc0jlb";
+  src = fetchFromGitHub {
+    owner = "Mikachu";
+    repo = "openbox";
+    rev = "release-${version}";
+    sha256 = "0k9ad731xf3l1zi3mf6ac80pal9azz4siwsv1zzg9qniksm2v0fn";
   };
 
+  patches = [
+    # Add python3 support
+    (fetchpatch {
+      name = "openbox-python3";
+      url = "https://github.com/Mikachu/openbox/pull/9.patch";
```

Comment:
This is not a stable url. It may change contents in the case of force push (unlikely in this case, but still).
Could fetch individual commits:

* https://github.com/Mikachu/openbox/commit/acfbbc4ea40932f183617bb7006700140fe5f61e.patch
* https://github.com/Mikachu/openbox/commit/bd0ca516f57a7ee6d43c663d09a275953ce56b4e.diff

or just the first patch from AUR

* https://git.archlinux.org/svntogit/community.git/plain/openbox/trunk/py3.patch?id=90cb57ef53d952bb6ab4c33a184f815bbe1791c0

Reply:
Used the AUR one since that one now correctly applies to the tarball.